services/agora/README.md

```diff
@@ -38,11 +36,15 @@ There is an example file `env.example` which can be used as the reference.
 
 ```bash
 cp env.example .env
-# modify .env
-. ./.env
-yarn dev
 ```
 
+Make sure that VITE_IS_ORG_IMPORT_ONLY="false"
```

Comment:
I think this should be removed.
Instead we should update `env.example`and add it, if it's not there already